src/util/Constants.js

```diff
@@ -49,6 +49,10 @@ exports.Endpoints = {
         if (dynamic) format = hash.startsWith('a_') ? 'gif' : format;
         return makeImageUrl(`${root}/avatars/${userId}/${hash}`, { format, size });
       },
+      GuildMemberAvatar: (guildId, memberId, hash, format = 'webp', size, dynamic = false) => {
+        if (dynamic) format = hash.startsWith('a_') ? 'gif' : format;
+        return makeImageUrl(`${root}/guilds/${guildId}/users/${memberId}/avatars/${hash}`, { format, size });
```

Comment:
Wouldn't the endpoint be `members` instead of `users` ?

Reply:
Nope, refer to @GamingGeek’s review above: https://cdn.discordapp.com/guilds/692925064135311390/users/90339695967350784/avatars/67fcd6af88d76888b8525c5c3684b6b2.png?size=2048